.github/workflows/on_PR_linux_matrix.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: install dependencies
         run: |
           sudo apt install -y tree
-          python3 -m pip install conan==1.59.0 ninja
+          python3 -m pip install conan==1.62.0 ninja
```

Comment:
Shall we just change all of these to `~=1.62` or `==1.*` and be done with it? This constant extract version babysitting is kinda annoying...

https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release

Reply:
Done.